tailwind.config.ts

```diff
@@ -34,9 +38,19 @@ const config = {
         'accordion-down': 'accordion-down 0.2s ease-out',
         'accordion-up': 'accordion-up 0.2s ease-out',
       },
+      fontSize: {
+        'hero-title': '2rem',
+        'hero-title-md': '2.625rem',
+        'hero-title-lg': '3rem',
+      },
```

Comment:
🟢nitpick: since the class for the hero font size is only used in one component, I wouldn't create a dedicated Tailwind class in the config file (because this file might grow too much when the app grows), I'd rather write custom CSS for it. But I'm not sure what's considered "industry standard" in this case or if there are even broadly accepted guidelines for this...